api/src/Entity/Activity.php

```diff
@@ -24,7 +24,7 @@
 #[ApiResource(
     collectionOperations: [
         'get' => [
-            'normalization_context' => ['groups' => ['read', 'Activity:ActivityResponsibles', 'Activity:ContentNodes']],
+            'normalization_context' => ['groups' => ['read', 'Activity:ActivityResponsibles']],
```

Comment:
This was added to make the react-print load time bearable during development. So we should soon add similar replacements for this to react-print as you did for nuxt-print.

Reply:
Yes, absolutely. I suggest we merge this first, and then we can use the same data loading concept for react-print (by me or you - both ok for me).